action/protocol/staking/stakingpb/staking.proto

```diff
@@ -36,9 +36,9 @@ message Candidate {
     string ownerAddress = 1;
     string operatorAddress = 2;
     string rewardAddress = 3;
-    bytes name = 4;
-    bytes votes = 5;
-    bytes selfStake = 6;
+    string name = 4;
+    string votes = 5;
```

Comment:
i feel bytes are more natural for *big.Int
but it's OK

Reply:
https://golang.org/pkg/math/big/#Int.Bytes
`Bytes returns the absolute value of x as a big-endian byte slice.`
i understand we store uint there but I would like to keep the real number just in case